tests/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixerTest.php

```diff
@@ -128,6 +128,10 @@ function Foo(INTEGER $a) {}
                 '<?php function Foo(object $A): void {}',
                 '<?php function Foo(OBJECT $A): VOID {}',
             ],
+            [
+                '<?php return function (callable $c) {};',
+                '<?php return function (CALLABLE $c) {};',
+            ],
```

Comment:
```suggestion
            [
                '<?php return function (callable $c) {};',
                '<?php return function (CALLABLE $c) {};',
            ],
            [
                '<?php return fn (callable $c): int => 1;',
                '<?php return fn (CALLABLE $c): INT => 1;',
            ],
```

I believe this fixer does not touch returned values, that's why I used `1` here. But argument's type and the return type should be fixed.

Reply:
Support for arrow functions is requested in #6632, I totally forgot about it 😅. We can close this thread.